src/modules/folder/folder.usecase.ts

```diff
@@ -55,6 +61,16 @@ export class FolderUseCases {
     return folder;
   }
 
+  async getFolderByUuid(folderUuid: FolderAttributes['uuid']): Promise<Folder> {
```

Comment:
This method already [exists](https://github.com/internxt/drive-server-wip/blob/master/src/modules/folder/folder.usecase.ts#L41). 

Also, I do not see this function being used on any of the introduced changes. If it is going to be used, use the already existing one.

If this is needed for accessing a folder by an invited user, you should: 
- Create a use case on private sharing folder use-cases (`getFolderByInvitedUser` p.e) where you check that the user has the proper permissions to access it, and then you pass to the already existent function here, the owner user object.